django/forms/models.py

```diff
@@ -1194,7 +1194,11 @@ def _get_queryset(self):
         return self._queryset
 
     def _set_queryset(self, queryset):
-        self._queryset = queryset
+        if queryset is not None:
+            # _queryset must be a QuerySet, not Manager
```

Comment:
Not sure if the comment is needed but a few teaks if so: `# _queryset must be a QuerySet, not a Manager.`

Reply:
The manager use case is not documented, that's why I included it. Since the test coverage is good enough, I agree that this comment can be removed.